--- htop-0.6.6/htop.desktop	2007-05-17 19:03:51.000000000 +0000
+++ htop-0.6.6/htop.desktop	2007-10-15 19:49:18.000000000 +0000
@@ -4,9 +4,11 @@
 Name=Htop
 Type=Application
 Comment=Show System Processes
+Comment[pl]=Wyświetlanie aktualnego stanu procesów
 Terminal=true
 Exec=htop
-Path=
 Icon=htop
-Categories=ConsoleOnly;System;Application;
+Categories=ConsoleOnly;System;Monitor;
 GenericName=Process Viewer
+GenericName[pl]=Przeglądarka procesów
+# vi: encoding=utf-8
